packages/create-canon/scaffold/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "canon-app",
+  "description": "A new instance of a canon app.",
+  "version": "1.0.0",
+  "private": true,
+  "main": "./index.js",
+  "scripts": {
+    "dev": "canon-dev",
+    "build": "canon-build",
+    "locale": "canon-locale"
```

Comment:
I know it's not specifically a canon script, but don't you think it's worth having the start script in here as well?

`"start": "node index.js"`

Reply:
ah, yeah, definitely goes here